core/orchestrator.go

```diff
@@ -97,23 +60,8 @@ func (orch *orchestrator) TranscoderSecret() string {
 	return orch.node.OrchSecret
 }
 
-func (orch *orchestrator) StreamIDs(job *ethTypes.Job) ([]StreamID, error) {
-	streamIds := make([]StreamID, len(job.Profiles))
-	sid := StreamID(job.StreamId)
-	vid := sid.GetVideoID()
-	for i, p := range job.Profiles {
-		strmId, err := MakeStreamID(vid, p.Name)
-		if err != nil {
-			glog.Error("Error making stream ID: ", err)
-			return []StreamID{}, err
-		}
-		streamIds[i] = strmId
-	}
-	return streamIds, nil
-}
-
-func (orch *orchestrator) TranscodeSeg(job *ethTypes.Job, ss *SignedSegment) (*TranscodeResult, error) {
-	return orch.node.TranscodeSegment(job, ss)
+func (orch *orchestrator) TranscodeSeg(md *SegmentMetadata, seg *stream.HLSSegment) (*TranscodeResult, error) {
```

Comment:
A little nitpicky, but I find it confusing to have `TranscodeSeg`, `transcodeSeg`, and `TranscodeSegment` that seem to have no real functional relationships (Normally, a `TranscodeSeg` would call `transcodeSeg` because of public/private function).  Can we come up with more appropriate function names?

Reply:
I do agree that the `TranscodeSegment` here is perhaps misattributed as public when it's really a private function.

* `TranscodeSeg` The "friendly" name of the external interface that's exposed to RPC, testing, etc.
* `TranscodeSegment` Agreed that this is misattributed as public when it's really a private function. Perhaps `sendToTranscodeLoop"` ?
* `transcodeSeg` This is **the** function that calls the transcoder. The naming seems OK here, but open to other ideas.

Reply:
Done